gut_to_soil_manuscript_figures/_methods.py

```diff
@@ -44,12 +44,11 @@ def pcoa_2d(output_dir: str, metadata: qiime2.Metadata,
     swap_axes = str(swap_axes)
     himalaya = str(himalaya)
     pit_toilet = str(pit_toilet)
-    export_legend = str(export_legend)
+    export_legend_str = str(export_legend)
```

Comment:
```suggestion
    export_legend = str(export_legend)
```
So the only thing this is doing is changing the variable name - the functionality isn't changing (since this was already being cast to a string). I triple checked, and this was evaluating to True/False as expected (prior to these changes).